docs/cinder-csi-plugin/using-cinder-csi-plugin.md

```diff
@@ -43,6 +44,52 @@ For each kubernetes official release, there is a corresponding release of Cinder
 
 For sidecar version compatibility with kubernetes, please refer [Compatibility Matrix](https://kubernetes-csi.github.io/docs/sidecar-containers.html) of each sidecar.
 
+## Driver Deployment
+
+You can either use the manifests under `manifests/cinder-csi-plugin` or the Helm chart `charts/cinder-csi-plugin`.
+
+### Command-line arguments
+
+In addition to the standard set of klog flags, `cinder-csi-plugin` accepts the following command-line arguments:
+
+<dl>
+  <dt>--nodeid &lt;node id&gt;</dt>
+  <dd>
+  This argument is required.
+
+  An identifier for the current node which will be used in OpenStack API calls.  This can be either the UUID or name of the OpenStack server, but note that if using name it must be unique.
+  </dd>
+
+  <dt>--endpoint &lt;endpoint&gt;</dt>
+  <dd>
+  This argument is required.
+
+  The endpoint of the gRPC server agents will use to connect to this CSI plugin, typically a local unix socket.
+
+  The manifests default this to `unix://csi/csi.sock`, which is supplied via the `CSI_ENDPOINT` environment variable.
+  </dd>
+
+  <dt>--cloud-config &lt;config file&gt; [--cloud-config &lt;config file&gt; ...]</dt>
+  <dd>
+  This argument must be given at least once.
+
+  The path to a driver config file. The format of this file is specified in [Driver Config](#driver-config).
+
+  If multiple configuration files are supplied they will be merged. In the case of a conflict, the last supplied config file will take precedence.
+
+  The manifests default this to `/etc/config/cloud.conf`, which is supplied via the `CLOUD_CONFIG` environment variable.
+  </dd>
+
+  <dt>--cluster &lt;cluster name&gt;</dt>
+  <dd>
+  This argument is optional.
+
+  The identifier of the cluster that the plugin is running in.
+
+  This will be added as metadata to every Cinder volume created by this plugin.
+  </dd>
+</dl>
+
```

Comment:
Could you convert those using markdown? The pure HTML would make it harder to maintain.

Reply:
I can't, because AFAICT this is the 'supported' way to do description lists. I don't like it either, but I couldn't find anything better.

Do you know a better way to do this?

Reply:
https://raw.githubusercontent.com/kubernetes/cloud-provider-openstack/master/docs/openstack-cloud-controller-manager/using-openstack-cloud-controller-manager.md

Search `enable-ingress-hostname`